Makefile

```diff
@@ -220,9 +249,8 @@ test:
 core-test: core
 	# The test executable has a few options that can be useful
 	# in some contexts.
-	# The following command ensures that we can call 'test.exe --help'
-	# without having to chdir into the test data folder.
-	./_build/default/tests/test.exe --show-errors --help 2>&1 >/dev/null
+	dune build ./_build/default/src/tests/test.exe
+	cd ./_build/default/src/tests/ && ./test.exe --show-errors --help 2>&1 >/dev/null
```

Comment:
could we instead run ./_build/default/src/tests/test.exe from the root, so that we would not need all those ../../../../ ?

Reply:
the issue is that `dune runtest` two lines below executes `test.exe` within the `_build/default/src/tests` directory. but I'm looking into whether we can change that.

Reply:
done in fe26ede0bb57f7f7269bf3705eda2f573c53bc36